lightning/src/ln/channelmanager.rs

```diff
@@ -11094,10 +11232,20 @@ where
 		None
 	}
 
-	fn release_held_htlc(&self, _message: ReleaseHeldHtlc) {}
+	fn release_held_htlc(&self, _message: ReleaseHeldHtlc, _context: AsyncPaymentsContext) {
```

Comment:
nit: Might better to start these methods with `handle_` given "release" is overloaded in `release_pending_messages`.

Reply:
That makes sense to me, will do this in the follow-up though since this is pre-existing and PR big.